src/Avalonia.Native/avn.idl

```diff
@@ -810,6 +810,7 @@ interface IAvnTrayIcon : IUnknown
     HRESULT SetIcon(void* data, size_t length);
     HRESULT SetMenu(IAvnMenu* menu);
     HRESULT SetIsVisible(bool isVisible);
+    HRESULT SetToolTipText(char* text);
```

Comment:
@kekekeks Is it OK to use `char*` in this case? I'm unclear about when it's safe to use `char*` vs `IAvnString` in the IDL.

Reply:
Explanation from @kekekeks:

> IAvnString is required when you are returning a string and memory needs to be released. If you are passing a string and it's never going to have null character inside, it's ok to use char*. Microcom handles conversion of `string` to UTF8

So all good here!